spec/rails_helper.rb

```diff
@@ -1,16 +1,18 @@
-ENV["RAILS_ENV"] ||= 'test'
+ENV['RAILS_ENV'] ||= 'test'
 require 'spec_helper'
-require File.expand_path("../../config/environment", __FILE__)
+require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
-require 'helpers/features_spec_helper'
 
-# require all config files in support folder
+# require all config/helpers files in support folder
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
Better to use Rails.root.join with separated arguments, for compatibility on various OS:

```
Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
```

This way allows Ruby to use `/` or `\` for path.